### PR TITLE
Add back button to results page

### PR DESCRIPTION
**Pull request by DilwoarH**
## What
 
Add back button to results page - when a user clicks it then they go back to the enter your postcode page

## Why

Standard navigation 

https://trello.com/c/ZiMAyVy1/858-add-back-button-to-results-page

:warning: This application is Continuously Deployed: :warning:

- Merged changes are automatically deployed to staging and production.

- Make sure you follow [the guidance for deployments](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) **before** you merge.

- Check your branch is being deployed in the [Release app](https://release.publishing.service.gov.uk/applications/collections), after merging.

### CODE REVIEW

File: app/views/coronavirus_local_restrictions/results.html.erb [hunk 1]
```diff
@@ -1,3 +1,9 @@
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", {
+    href: find_coronavirus_local_restrictions_path
+  } %>
+<% end %>
```
Comment (leenagupte) [inline app/views/coronavirus_local_restrictions/results.html.erb:5]: This needs to be added to the "no results" page as well:

Screenshot 2020-10-26 at 11 15 01

**Review by leenagupte — COMMENTED** [review]
Looks good! I've tested locally and it all seems to work.

The back link just needs to be added to the "no results" page. See inline comment for a screenshot.

File: app/views/coronavirus_local_restrictions/results.html.erb (hunk 1 above)
Comment (DilwoarH) [reply app/views/coronavirus_local_restrictions/results.html.erb:5]: Done

**Review by leenagupte — APPROVED** [review]
👍 🎉

**Pull request merged**